### PR TITLE
Sachdevavaibhav/issue#4945

### LINKED ISSUE #4945: Replace Material UI components in `src/Components/Facility/AssetCreate.tsx`:
```tsx
10: import CropFreeIcon from "@material-ui/icons/CropFree";\n20: import { LocationOnOutlined } from "@material-ui/icons";
```

### Tasks

Ensure all the below tasks are accomplished if applicable to the file being modified in the PR.

- [ ] Migrate all Material UI components to use new Care UI / custom components. (ref: [🛠️ Tailwinding Care](https://github.com/coronasafe/care_fe/issues/3742))
- [ ] Migrate all Material UI icons to use [🏥 **Care Icons**](https://github.com/coronasafe/care_fe/pull/4071).
- [ ] Migrate `PageTitle` to `Page` component if any. (See [example](https://github.com/coronasafe/care_fe/pull/4796/files#diff-8e39f154ccf13c9dedd0f968d21a0abbde450e88df9724a4e7d4264d487a7846))

For more information see the epic issue that tracks this issue.

### PR DESCRIPTION
**Pull request by sachdevavaibhav**
### WHAT
Replaced legacy components from AssetCreate.tsx

## Proposed Changes

- Fixes #4945 
- Replaced LegacyActionTextInputField with TextFormField
- Replaced LegacyErrorHelperText with FieldErrorText

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
You should also be replacing html components such as `textarea` and all with respective FormField Components too. 

You've also not changed `PageTitle` to `Page` component either.

**Comment by sachdevavaibhav** [comment]
@rithviknishad I have made the changes. Kindly review it.

File: src/Components/Facility/AssetCreate.tsx [hunk 1]
```diff
@@ -385,32 +385,33 @@ const AssetCreate = (props: AssetProps) => {
   if (locations.length === 0) {
     return (
       <div className="px-6 pb-2">
```
Comment (rithviknishad) [inline src/Components/Facility/AssetCreate.tsx]: Again, no padding or margins to be applied above the `Page` component

File: src/Components/Facility/AssetCreate.tsx [hunk 2]
```diff
@@ -460,7 +461,7 @@ const AssetCreate = (props: AssetProps) => {
 
   return (
     <div className="pb-2 relative flex flex-col">
```
Comment (rithviknishad) [inline src/Components/Facility/AssetCreate.tsx]: No padding or margins to be applied above the Page component

File: src/Components/Facility/AssetCreate.tsx [hunk 3]
```diff
@@ -473,434 +474,417 @@ const AssetCreate = (props: AssetProps) => {
...
-                    />
+                    <div className="mt-6" />
+                    {sectionTitle("Service Details")}
+
+                    {/* Last serviced on */}
+                    <div
+                      className="col-span-6 sm:col-span-3"
+                      ref={fieldRef["last_serviced_on"]}
+                    >
+                      <label htmlFor="last-serviced-on">Last Serviced On</label>
```
Comment (rithviknishad) [inline src/Components/Facility/AssetCreate.tsx]: Why is this label here? We can pass it as label prop of the below textformfield ryt?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Facility/AssetCreate.tsx (hunk 3 above)
Comment (sachdevavaibhav) [reply src/Components/Facility/AssetCreate.tsx]: Yes, you are right. I have fixed this. Kindly review the changes.

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Comment by nihal467** [comment]
@sachdevavaibhav can you resolve the merge conflict

**Comment by sachdevavaibhav** [comment]
@nihal467 Ya sure!

**Comment by sachdevavaibhav** [comment]
@nihal467 Solved the merge conflict. Kindly review it.

**Comment by nihal467** [comment]
LGTM

**Pull request merged**